core/commands/storage.go

```diff
@@ -438,6 +438,7 @@ Upload a file with offline signing. I.e., SDK application acts as renter.`,
 			return err
 		}
 
+		output.ss.SetFileSize(fileSize) // set session file size
```

Comment:
The reason this was missed was because it's not part of the params for `openSession`. Should be handled inside there, so this line doesn't need to be written twice (not a good fix in the first PR)

Reply:
ok. working on